package.json

```diff
@@ -53,7 +53,7 @@
     "tslint": "^4.0.0",
     "tslint-config-ktsn": "^2.0.0",
     "typescript": "^2.1.4",
-    "vue": "^2.1.10",
+    "vue": "^2.2.6",
```

Comment:
Why do we need 2.3.0 or later?
I have no idea about changes that breaks Sinai in Vue 2.3.0...
Also, if we drop 2.2.6 or former, this should be `^2.3.0` shoundn't it?